src/components/Notifications/NotificationsBehaviorGroupTable.tsx

```diff
@@ -188,6 +203,64 @@ const getActions = (
   };
 };
 
+export const NotificationsBehaviorGroupRow: React.FunctionComponent<
```

Comment:
This file is getting to be pretty long. Could we extract the NotificationsBehaviorGroupRow component into its own file?

Reply:
Sounds good! Moved into [NotificationsBehaviorGroupRow.tsx](https://github.com/RedHatInsights/notifications-frontend/blob/d9db4247cd00e08c9ecbe9ef2b52b0c4bc50b055/src/components/Notifications/NotificationsBehaviorGroupRow.tsx)